site/content/docs/5.1/components/offcanvas.md

```diff
@@ -151,7 +151,7 @@ Responsive offcanvas classes hide content outside the viewport from a specified
 <div class="offcanvas-lg offcanvas-end" tabindex="-1" id="offcanvasResponsive" aria-labelledby="offcanvasResponsiveLabel">
   <div class="offcanvas-header">
     <h5 class="offcanvas-title" id="offcanvasResponsiveLabel">Responsive offcanvas</h5>
-    <button type="button" class="btn-close" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+    <button type="button" class="btn-close" data-bs-dismiss="offcanvas" data-bs-target="#offcanvasResponsive" aria-label="Close"></button>
```

Comment:
Sorry to disturb an old PR again, but I don't really understand why this change was needed ? I mean I understand that without it, it doesn't work anymore, but I can't explain why. Could you light me up about this ?

Reply:
I don't know either actually, at least in hindsight lol. @GeoSot Any ideas?